packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ipv6/ipv6.controller.js

```diff
@@ -43,6 +50,9 @@ export default class AgoraIpV6OrderController {
     this.ipv6RegionsWithPlan = this.getIpv6RegionsWithPlan();
     this.canOrderIpv6 = true;
     this.regionState = {};
+
+    this.trackClick();
```

Comment:
The click action : dedicated::ip::dashboard::order should probably be made here : https://github.com/ovh/manager/blob/8442c678312f7f7df779530313bb153b8f2ef5c4/packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ip-ip-agoraOrder.html#L69

Reply:
for this action the traking already exist  here https://github.com/ovh/manager/blob/749064519e9b36712552a5b4cf1bedaaf9f33b49/packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ipv6/ipv6.routes.js#L11 with navigate traking type
and this one like as ipv4  https://github.com/ovh/manager/blob/ba3910bd07a9be33714d690cea90d4dccd95ddb3/packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ipv4/ipv4.controller.js#L88

Reply:
I agree with Quentin. It doesn't make sense to add it inside init function then adding the trackClick() in https://github.com/ovh/manager/blob/feat/MANAGER-13146/packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ip-ip-agoraOrder.controller.js#L78